src/lib/style/components/_personal-access-tokens.scss

```diff
@@ -18,10 +18,7 @@
   .SRC-scopeName {
     margin-inline-start: 3px;
     margin-inline-end: 3px;
-  }
-
-  .SRC-scopeName:hover {
-    text-decoration: underline;
+    cursor: help;
```

Comment:
Do the scope names act like normal checkbox text (you can click to select the scope)?  

The **help** cursor type is not a commonly-used interface anymore.
We don't even use it for our contextual help widgets (because we want to indicate that they're clickable).

Reply:
These scope names are not in the modal (there's no checkbox). They appear on the cards of tokens that are already created. When you hover over a scope, you get a tooltip describing the scope. The issue was that the scope names appear clickable because of the underline on hover, but there's no click behavior.

I can remove the cursor type entirely; just not sure how to demonstrate that these pieces are interactive, but not clickable

Reply:
I think setting it to **default** is preferable for this case.

Reply:
(not **pointer**, which would indicate that it's clickable.)

Reply:
Communicating that they are interactive should not be the responsibility of the cursor, since by the time you see the cursor you see that it's interactive (from the hover tooltip!).

Reply:
Makes sense! I've set the cursor to default on hover.